cdap-ui/app/services/constants.js

```diff
@@ -76,7 +76,8 @@ angular.module(PKG.name + '.services')
           parallelConnectionError: 'Please remove parallel connections in this pipeline.',
           pluginDoesNotExist: 'This plugin does not exist: ',
           unsavedPluginMessage1: 'There are unsaved changes for node \'',
-          unsavedPluginMessage2: '\'. Please save them before publishing the pipeline.'
+          unsavedPluginMessage2: '\'. Please save them before publishing the pipeline.',
+          pipelineNameAlreadyExistError: 'A pipeline with this name already exists. Please choose a different name.'
```

Comment:
Question: In Javascript, can the last element in an array end with a comma? If so, by always adding a comma after the last item, when other items are added, you only change one line and not two. Makes reviews easier. Python allows that, and so you often see Python code such as:
```
my_list = [ 'a', 
    'b',
    ]
```
so that ``c`` can be added as ``'c',``...